app/Http/Middleware/LimitWikiAccess.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+use Symfony\Component\HttpFoundation\Response;
+use App\WikiManager;
+
+class LimitWikiAccess
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Closure(\Illuminate\Http\Request): (\Symfony\Component\HttpFoundation\Response)  $next
+     */
+    public function handle(Request $request, Closure $next): Response
+    {
+        $userHasAccess = WikiManager::where([
+            'user_id' => $request->user()?->id,
+            'wiki_id' => $request->input('wiki'),
```

Comment:
do we need validation/sanitation here for the two user inputs?

Reply:
I would really hope Laravel does not let me pass user provided SQL to a QueryBuilder, but just for the sake of better error messages validation probably makes sense. Will add this.

Reply:
Added this :heavy_check_mark:

Reply:
> I would really hope Laravel does not let me pass user provided SQL to a QueryBuilder

I'd be quite surprised if Laravel would circumvent that actually (not that it wouldnt be a great idea, I just dont have high hopes here :D)

Reply:
I think unless you use `raw` methods, this is prevented by Eloquent https://www.stackhawk.com/blog/sql-injection-prevention-laravel/